src/services/auth.js

```diff
@@ -1,40 +1,41 @@
-const bcrypt = require("bcrypt");
+const bcrypt = require('bcrypt');
+
 const saltRounds = 10;
-const jwt = require("jsonwebtoken");
-const {jwtSecret} = require("./config");
+const jwt = require('jsonwebtoken');
+const { jwtSecret } = require('./config');
```

Comment:
could we have the requires together and `saltRounds` separate?

Reply:
yupp